site/docusaurus.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     favicon: 'img/favicon.ico',
     organizationName: 'pure-css',
     projectName: 'pure-css.github.io',
```

Comment:
This needs to be in metaData otherwise docusaurus won’t work. You can run the site locally to test it out.

Reply:
I meant the repoName property. Custom properties have to be in metaData.

Reply:
Oops, my bad, will fix asap.